trimmer.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'base'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    name = @nameable.correct_name
+    if name.length > 10
+      name[0, 10]
+    else
+      name
+    end
```

Comment:
[REQUIRED] Kindly fix the logic for your code. Don't forget that Ruby uses zero indexing so we should stop at `9` not `10`.